api/src/main/java/org/pmiops/workbench/api/AppsController.java

```diff
@@ -64,7 +64,7 @@ public ResponseEntity<EmptyResponse> deleteApp(
   }
 
   @Override
-  public ResponseEntity<App> getApp(String workspaceNamespace, String appName) {
+  public ResponseEntity<UserAppEnvironment> getApp(String workspaceNamespace, String appName) {
```

Comment:
New name for the API entity: UserApp**Environment** because this describes a specific **instance** of a user app.

Should we also rename this method, and similar ones? `getAppEnvironment()` or `getUserAppEnvironment()` perhaps

Reply:
Could also rename this controller ... but I think the fact that it's plural (**Apps**Controller) makes it clear already.  But WDYT?